src/Compilers/CSharp/Portable/Emitter/Model/SynthesizedPrivateImplementationDetailsStaticConstructor.cs

```diff
@@ -21,7 +24,34 @@ internal override void GenerateMethodBody(TypeCompilationState compilationState,
         {
             SyntheticBoundNodeFactory factory = new SyntheticBoundNodeFactory(this, this.GetNonNullSyntaxNode(), compilationState, diagnostics);
             factory.CurrentMethod = this;
-            
+
+            ArrayBuilder<BoundStatement> body = ArrayBuilder<BoundStatement>.GetInstance();
+
+            // Initialize the payload arrays for each kind of dynamic analysis instrumentation.
+            // For each kind of instrumentation:
+            //
+            //     payloadArray = new T[MethodDefinitionTokenIndex + 1][];
```

Comment:
This is the C# syntax for creating an array of arrays. I'll rephrase the comment to make it clear that what is being allocated is an array of arrays, but I can't improve the C# syntax.